contracts/src/challenge/SymChallenge.sol

```diff
@@ -149,6 +191,8 @@ contract SymChallenge is ChallengeBase, ISymChallenge {
         emit Bisected(bisectionHash, challengedSegmentStart, challengedSegmentLength);
     }
 
+    // TODO: If bisection is not executed because both have commited to the same has at the same `numStep`, verify
+    //        there are no additional ops on the stack after the last step.
```

Comment:
typo in `"same has"` (i assume you mean hash) + indentation looks off

Reply:
Fixed.